<?xml version="1.0"?>

<launch>

    <arg name="bootstrap_server" default="114.70.21.162:9092"/>
    <arg name="kafka_topic" default="external-comm-test"/>
    <arg name="ros_topic" default="/ublox_gps/fix"/>
    <arg name="msg_type" default="sensors_msgs/NavSatFix"/>

    <node pkg="ros_kafka_connector" type="kafka_publish.py" name="kafka_publish">
        <param name="~bootstrap_server" value="$(arg bootstrap_server)"/>
        <param name="~kafka_topic" value="$(arg kafka_topic)"/>
        <param name="~ros_topic" value="$(arg ros_topic)"/>
        <param name="~msg_type" value="$(arg msg_type)"/>
    </node>

</launch>
